lib/WeBWorK/PG/IO.pm

```diff
@@ -131,7 +132,8 @@ sub read_whole_problem_file {
 
 sub read_whole_file {
 	my $filePath = shift;
-
+	warn "<br/>Can't read file $filePath<br/>" unless -r $filePath;
```

Comment:
Why is there the initial <br> here?  It makes the error message look a little wierd.